configurations/performance/cql-stress-650gb-8-col-i4i-mini-test-throughput-oss.yaml

```diff
@@ -0,0 +1,19 @@
+prepare_write_cmd: [ "cql-stress-cassandra-stress write no-warmup cl=ALL n=6500000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=500 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1..6500000",
```

Comment:
How this configuration is tied to the `OSS`  part in the file name?

Reply:
it's just a copy of oss file (as ent and oss differ in throttling).
Maybe for this test it doesn't matter as it's not official, but why caring?

Reply:
just leave a comment inside of it, to point which configuration it's based on.

you can drop `oss` in the process, it doesn't make any difference

Reply:
fixed